src/components/menu.js

```diff
@@ -27,50 +28,51 @@ const MenuContainer = styled.div.attrs({
   max-width: 1440px;
   height: 100vh;
   width: ${props => (props.active ? "100vw" : "0")};
-  transition: 0.8s;
-  transition-delay: 0.1s;
+  transition: 0.5s;
+  // transition-delay: 0.2s;
   top: 0;
-  z-index: 2;
+  left: 0;
+  z-index: 11;
 `
 
 const MenuSidebar = styled.div.attrs({
   className: "bg-yellow",
 })`
   height: 100vh;
   width: ${props => (props.active ? "30vw" : "0")};
-  transition: 0.8s;
-  transition-delay: 0.1s;
+  transition: 0.5s;
+  // transition-delay: 0.1s;
```

Comment:
are these just being left in commented out for now until we decide what animation we're going for?

Reply:
I think so yes :)